amplipi/streams.py

```diff
@@ -632,8 +648,126 @@ def __str__(self):
     mock = ' (mock)' if self.mock else ''
     return 'internetradio connect: {self.name}{connection}{mock}'
 
+class Plexamp:
```

Comment:
You should probably use  f'' format strings here, just to make the code a little more readable